modules/test/playwright/tests/commerce/commerceProducts.spec.ts

```diff
@@ -470,3 +465,233 @@ test('COMMERCE-8153 Verify the visibility rules', async ({
+								en_US: 'Value1',
+							},
+							priority: 1,
+							quantity: 1,
+							skuId: product1.skus[0].id,
+						},
+						{
+							deltaPrice: 0.0,
+							key: 'value2',
+							name: {
+								en_US: 'Value2',
+							},
+							preselected: true,
+							priority: 2,
+							quantity: 1,
+							skuId: product2.skus[0].id,
+						},
+					],
+					skuContributor: true,
+				},
+				{
+					fieldType: 'select',
+					key: 'option2',
+					name: {
+						en_US: 'Option2',
+					},
+					optionId: option2.id,
+					priceType: 'static',
+					priority: 2,
+					productOptionValues: [
+						{
+							deltaPrice: 0.0,
+							key: 'value3',
+							name: {
+								en_US: 'Value3',
+							},
+							priority: 1,
+							quantity: 1,
+							skuId: product3.skus[0].id,
+						},
+						{
+							deltaPrice: 0.0,
+							key: 'value4',
+							name: {
+								en_US: 'Value4',
+							},
+							preselected: true,
+							priority: 2,
+							quantity: 1,
+							skuId: product4.skus[0].id,
+						},
+					],
+					skuContributor: true,
+				},
+			],
+		});
+
+		await applicationsMenuPage.goToProducts();
+
+		await commerceAdminProductPage.managementToolbarSearchInput.fill(
+			'ProductBundle'
+		);
+		await commerceAdminProductPage.managementToolbarSearchInput.press('Enter');
+
+		await page.getByRole('link', {exact: true, name: 'ProductBundle'}).click();
+
+		await commerceAdminProductPage.generateSkus();
+
+		const account = await apiHelpers.headlessAdminUser.postAccount({
+			name: getRandomString(),
+			type: 'person',
+		});
+
+		apiHelpers.data.push({id: account.id, type: 'account'});
+
+		await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+			account.id,
+			['test@liferay.com']
+		);
+
+		await applicationsMenuPage.goToSite(site.name);
+
+		await commerceLayoutsPage.goToPages(false);
+		await commerceLayoutsPage.createWidgetPage(getRandomString());
+
+		await page.goto(`/web/${site.name}`);
+
+		await productDetailsPage.addProductDetailsWidget();
+
+		await page.goto(`/web/${site.name}/p/productbundle`);
+
+		await expect(page.getByText('Value2', {exact: true})).toBeVisible();
+		await expect(page.getByText('Value4', {exact: true})).toBeVisible();
+
+		await productDetailsPage.addToCartButton.click();
+
+		await expect(page.getByText('Danger:Product4 cannot be combined with Product2.')).toBeVisible();
+	}
+	finally {
+		await commerceAdminProductPage.gotoProduct(product2Name);
+
+		await commerceAdminProductDetailsProductRelationsPage.productRelationsLink.click();
+
+		await expect(
+			(
+				await commerceAdminProductDetailsProductRelationsPage.tableRow(
+					2,
+					product4Name,
+					true
+				)
+			).row
+		).toBeVisible();
+
+		await commerceAdminProductDetailsProductRelationsPage.selectItemsInput.check();
+
+		await expect(
+			commerceAdminProductDetailsProductRelationsPage.deleteBulkButton
+		).toBeVisible();
+
+		await commerceAdminProductDetailsProductRelationsPage.deleteBulkButton.click();
```

Comment:
Don't we have an api to delete the relationships that we can use instead of the UI?